pkg/handlers/authorization/handler.go

```diff
@@ -45,36 +47,25 @@ func (auth *authorizationHandler) Process(req *http.Request, context *handlers.R
 	context.Token = getBearerTokenFrom(req)
 	if context.Token == "" {
 		log.Debugf("Skipping %s as there is no bearer token present", auth.Name())
-		return req, nil
+		return req, errors.New("missing bearer token")
```

Comment:
use `fmt.Errorf` to construct the error, adding whatever context information, names, etc. to the error message

Reply:
The current message is alright, the is no context information at this point.